inshorts.py

```diff
@@ -0,0 +1,95 @@
+# inshorts module contains Inshorts class.
+# inshorts class will be used to fetch the news from inshorts.com as API response for it's instance.
+
+# standard library's
+import requests
+
+
+class Inshorts:
+    """
+    Inshorts module is the main module which will get data from Inshorts API
+    Inshorts class helps in providing 2 varities of News(all_news & Category_news).
+    """
+
+    url = "https://inshorts.com/api/en"
+
+    # def __init__(self,ob_name):
+    #     """Initialize Objects with it's name"""
+    #     self.ob_name = ob_name
+
+    def all_news(self, max_limits=5):
+        """Provides the all news via Inshorts API
+
+        Args:
+            max_limit: maximum limit of the selected news.
+        """
+
+        try:
+            url = f"{self.url}/news"
```

Comment:
Use **/news** as pre Defined Endpoints
bcz when incase of multiple uses of endpoints or similar endpoints things can become messy 
use standard like 
NEW_PATH  = "/news"
and use **furl** like lib for creating new URL paths instead of directly concatenating to URL.